### ISSUE TITLE
Looking for alpha testers

### ISSUE THREAD

**Issue by RomkeVdMeulen** [StoneAgeFactorio/StoneAgeFactorio#4]
The first alpha version is nearly done. It has a lot of placeholder graphics and sounds, but the gameplay should be pretty similar to the final product.

Now I'm looking for people who'd like to play-test it and give feedback.

**Comment by xJDavisx** [comment]
Played it for maybe 10-20 minutes or so.
I got to the point where I could make the campfire and the fire hardened sticks.

Issues I've noticed:
- Game lags a bit when you switch tools
- Item descriptions can be vague or not clear. I searched for a bit trying to find a resource that would give me vines, then I found out that you get them from trees when you cut them down with sharp sticks. Maybe that should be more clear?
- The sand resource didn't spawn enough to allow me to build more than 1 campfire. I searched for most of my time playing for more sand and gave up. Even restarted a few times.

Overall, I think the mod has promise. I might go back in a while to see if I can get farther. If I do, I'll post again or edit this one with anything else I find. Good work man!

**Comment by RomkeVdMeulen** [comment]
Thanks!

You're right: the descriptions could use more work. I concentrated on the content and didn't give the text much thought. I'll polish it up a bit.

There should be four sand patches spawned, though it can be very hard to find them. If you have a basket equiped, the deconstruction planner should highlight them. Making the patches more visible is definitely high on the to-do list.

**Comment by RomkeVdMeulen** [comment]
I've rewritten the descriptions to be more useful (e45e419): less fluf, more describing the effects and requirements of items. I'm open to suggestions for further improvements.

**Comment by RomkeVdMeulen** [comment]
Bedankt!

I've already switched the generation of sand and clay to the normal Factorio resource spawner, so starting with the next version there should be more of it and it should be easier to find. I'll look into making the furnaces able to burn charcoal, but I believe you can only specify one fuel category per building so that may not be possible. Besides, I want to make it so you have to use the kiln much more before you finish the mod: see #17.

**Comment by RomkeVdMeulen** [comment]
Burner mining drills, burner inserters, stone furnaces and boilers are now all able to burn charcoal.

**Comment by RomkeVdMeulen** [comment]
Release 0.0.2 is done! https://github.com/StoneAgeFactorio/StoneAgeFactorio/releases/tag/0.0.2

**Comment by leo1123** [comment]
I've played it a while, I think it should be more clear that you have to start with cutting down a dead tree, spent the first 10 minutes trying to find out how to get any resource at all.

**Comment by leo1123** [comment]
for everything else it is good, just an annoyance that you have to fuel a wooden resevoir but I know that it is impossible to fix until 0.17 of the game.

**Comment by RomkeVdMeulen** [comment]
Good point. I don't want to just tell the player outright to cut down a dead tree though, that would take away from the puzzle. But perhaps I can think of a better hint in the initial message, or add something to the sharpened stick description about where to get sticks.

**Comment by sf17k** [comment]
Tested, no major bugs. Some feedback:

I had assumed all trees were the same so when I came across a dead one I thought "this can't be the solution, can it?" The hint did help but I nearly skipped reading it and I'm not sure it'd be spoiling a puzzle to say something a caveman should already know. The very first task should be a tutorial, not a puzzle, right?

Finding clay was tricky, I thought it'd be a resource patch and couldn't find any on the minimap. I even looked for ways to mine it from the shoreline, but finally came across some.

A lot of the recipes are cheap, quick, and sort of "one-time" use, so I was going through technologies as fast as I could figure them out. Could there be more interesting gameplay to be had here? Simply increasing recipe difficulty would let me spend more time with the stone age tech. For example, mixing clay became a kinda interesting minigame cuz I had to do a lot of clicking. It just seems odd that I only ever build one campfire, one mixing pit, one charcoal pile, one forge, one anvil. If a single recipe took a really long time, I might build five forges to make ten iron blooms. Or if I had to gradually work my way up material quality, e.g. starting with a copper anvil and copper hammer, then iron anvil and iron hammer, then I get to spend more time with each of the mechanics.

TerraFirmaCraft is a Minecraft total conversion that also explores primitive technology. Among other things, it has a neat way of handling charcoal. You have to build up a pile of logs, cover them in dirt, light it, seal it, and wait a full day cycle for them to finish. It feels rewarding because it takes so long to set up and finish (chopping trees to create the log piles is also time consuming) and adds a bit of planning.

Speaking of charcoal, it wasn't obvious to me that I needed 5 logs for the recipe, so for a while I was confused why it wouldn't start. Seemed like a bug.

It's sad that I completely ditch all my stone age tech the moment I get an iron pickaxe, but it's nice that I can get rid of some of it by burning.

**Comment by RomkeVdMeulen** [comment]
About the smoother transition into the burner stage: I'd love to get people thoughts and suggestions on this. Please post them to #28.
